pyomo/core/base/units_container.py

```diff
@@ -33,22 +33,23 @@
 be used directly in expressions (e.g., defining constraints). You can also
 verify that the units are consistent on a model, or on individual components
 like the objective function, constraint, or expression using
-`assert_units_consistent`. There are other methods that may be helpful
-for verifying correct units on a model.
+`assert_units_consistent` (from pyomo.util.units_checking).
```

Comment:
Silly naming question, but would `pyomo.util.units_checker` be a better (more intuitive) name for the new module?

Reply:
to me, units_checker implies that there is an class (just because it is a noun, I guess). This module is currently only a list of functions, which is why I named it the way I did. I don't feel strongly about it.